cmd/publisher/commands/ui.go

```diff
@@ -11,13 +11,7 @@ import (
 )
 
 type UICmd struct {
-	Path          util.Path `help:"Path to project directory containing files to publish." arg:"" default:"."`
```

Comment:
I wouldn't remove the PATH from this command. I frequently use this to run the API from a specific base folder for testing purposes.

Reply:
For example I find this command very helpful: `just run ui test/sample-content/fastapi-simple --listen=localhost:9001 -vv`

Reply:
I guess that is easier than specifying a dir parameter in the API call, so I've added it back in!